Fix Format errors on cran

```r
Found the following significant warnings:
  xlsb_funs.h:1745:17: warning: format '%d' expects argument of type 'int', but argument 2 has type 'size_t' {aka 'long unsigned int'} [-Wformat=]
  xlsb_funs.h:1745:21: warning: format '%d' expects argument of type 'int', but argument 3 has type 'size_t' {aka 'long unsigned int'} [-Wformat=]
  xlsb_funs.h:1764:37: warning: format '%d' expects argument of type 'int', but argument 2 has type 'std::vector<int>::size_type' {aka 'long unsigned int'} [-Wformat=]
  xlsb_funs.h:1764:44: warning: format '%d' expects argument of type 'int', but argument 3 has type 'size_t' {aka 'long unsigned int'} [-Wformat=]
  xlsb.cpp:1683:21: warning: format '%d' expects argument of type 'int', but argument 2 has type 'size_t' {aka 'long unsigned int'} [-Wformat=]
  xlsb.cpp:1683:25: warning: format '%d' expects argument of type 'int', but argument 3 has type 'size_t' {aka 'long unsigned int'} [-Wformat=]
```

--- a/src/xlsb.cpp
+++ b/src/xlsb.cpp
@@ -1680,7 +1680,7 @@ int workbook_bin(std::string filePath, std::string outPath, bool debug) {
         }
 
         if ((size_t)bin.tellg() != end_pos) {
-          Rprintf("%d: %d", (size_t)bin.tellg(), end_pos);
+          Rprintf("%d: %d", (int)bin.tellg(), (int)end_pos);
           Rcpp::stop("repositioning");
         }
 

--- a/src/xlsb_funs.h
+++ b/src/xlsb_funs.h
@@ -1742,7 +1742,7 @@ std::string CellParsedFormula(std::istream& sas, bool swapit, bool debug, int co
     if (debug) Rcpp::Rcout << ".";
     if (debug) {
       Rprintf("Formula cb: %d\n", val1);
-      Rprintf("%d: %d\n", (size_t)sas.tellg(), pos);
+      Rprintf("%d: %d\n", (int)sas.tellg(), (int)pos);
     }
     // this is a little risky. maybe its some kind of vector indicating the
     // order in which extra elements are going to be selected?
@@ -1761,7 +1761,7 @@ std::string CellParsedFormula(std::istream& sas, bool swapit, bool debug, int co
         Rcpp::Rcout << ptgextra[cntr] << std::endl;
       }
     } else if (ptgextra.size() < (cntr + 1)) {
-      if (debug) Rprintf("ptgextra %d and %d\n", ptgextra.size(),  cntr);
+      if (debug) Rprintf("ptgextra %d and %d\n", (int)ptgextra.size(),  (int)cntr);
     }
 
 

